insights/parsers/etcd_conf.py

```diff
@@ -0,0 +1,70 @@
+"""
+ETCD Configuration Files
+========================
+Parsing and extracting data from output of config file ``/etc/etcd/etcd.conf``.
+
+Parsers provided by this module are:
+EtcdConf - file ``/etc/etcd/etcd.conf``
+---------------------------------------
+"""
+
+from .. import Parser, parser, get_active_lines
+from ..parsers import split_kv_pairs
+from insights.specs import Specs
+from insights.parsers import ParseException, SkipException
+
+@parser(Specs.etcd_conf)
+class EtcdConf(Parser):
+    """
+    Parser for configuration for ETCD.
+
+    Typical Output looks like::
+
+        ETCD_NAME=f05-h19-000-1029p.rdu2.scalelab.redhat.com
+        ETCD_LISTEN_PEER_URLS=https://10.1.40.235:2380
+        ETCD_DATA_DIR=/var/lib/etcd/
+        #ETCD_WAL_DIR=
+        #ETCD_SNAPSHOT_COUNT=10000
+        ETCD_HEARTBEAT_INTERVAL=500
+        ETCD_ELECTION_TIMEOUT=2500   
```

Comment:
Have you considered using the [IniConfigFile parser](https://insights-core.readthedocs.io/en/latest/api_index.html#insights.core.IniConfigFile) instead of Parser?  It looks like based on the test data it would be a better choice.

Reply:
Thanks for the pointer, @bfahr. I've used the IniConfigFile Parser and made the changes. Pushing the revised code now :)